### PR TITLE
feat: channel for phone login consent

### PR DESCRIPTION
**Pull request by siddhart1o1**

### CODE REVIEW

File: apps/consent/services/galoy-auth/index.ts [hunk 1]
```diff
@@ -16,13 +16,7 @@ const authApi = {
...
   },
 
-  validateTotp: async (
-    totpCode: string,
-    authToken: string,
-    customHeaders?: object,
-    /* eslint @typescript-eslint/ban-ts-comment: "off" */
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  ): Promise<any> => {
+  validateTotp: async (totpCode: string, authToken: string, customHeaders?: object) => {
```
Comment (dolcalmi) [inline apps/consent/services/galoy-auth/index.ts:19]: please use 1 param rule (required for tracing)

File: apps/consent/app/login/phone/server-actions.ts [hunk 2]
```diff
@@ -24,11 +25,18 @@ export const getCaptchaChallenge = async (
   const login_challenge = form.get("login_challenge")
```
Comment (dolcalmi) [inline apps/consent/app/login/phone/server-actions.ts:25]: when did we begin to use snake case? please use always camel case

File: apps/consent/app/components/select.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  label?: string
+  id: string
+  options: string[]
+}
+
+const SelectComponent: React.FC<SelectProps> = ({
+  label,
+  id,
+  options,
+  ...selectProps
```
Comment (dolcalmi) [inline apps/consent/app/components/select.tsx:13]: please validate with nicolas, I dont think this is a good practice

**Review by dolcalmi — COMMENTED** [review]
few comments... like I mentioned yesterday this project is accumulating a lot of technical debt

**Review by dolcalmi — APPROVED** [review]

**Comment by sandipndev** [comment]
@dolcalmi is it possible to add these rules in via eslint?

**Pull request merged**